src/client/components/layout.js

```diff
@@ -145,25 +144,25 @@ class Layout extends React.Component {
 				}
 				{!(homepage || hideSearch) &&
 				<Navbar.Form pullRight
-					action="/search"
-					role="search"
+					 action="/search"
+					 role="search"
 				>
-					<FormGroup>
+					<div className="form-group">
```

Comment:
Could you explain a little bit more about this? Add a comment explaining why it's not good to use FormGroup here, so that we'll know why in a few years when we come back to this code.